TombEngine/Game/effects/tomb4fx.cpp

```diff
@@ -1479,7 +1479,7 @@ int GetFreeShockwave()
 	return -1;
 }
 
-void TriggerShockwave(Pose* pos, short innerRad, short outerRad, int speed, unsigned char r, unsigned char g, unsigned char b, unsigned char life, short angle, short flags)
+void TriggerShockwave(Pose* pos, short innerRad, short outerRad, int speed, unsigned char r, unsigned char g, unsigned char b, unsigned char life, short angle, short damage)
```

Comment:
`TriggerShockwave` function declaration should accept `damage` as `ShockwaveDamage` too. Underlying `SHOCKWAVE_STRUCT` may also need to convert `flags` field to `ShockwaveDamage` enum class, and also rename `flags` to `damage`.

You may also consider doing the opposite, and save precise damage value in `flags` field (which also should be renamed in this case). In such scenario, no `ShockwaveDamage` will be needed.